omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleActionResource1_8.java

```diff
@@ -109,12 +113,32 @@ public Object create(SimpleObject post, RequestContext context) throws ResponseE
 				case UNLOAD:
 					unloadModules(modules, servletContext);
 					break;
+				case UPDATE:
+					updateModules(modules, action.getDownloadUrls());
+					break;
 			}
 		}
 		
 		return ConversionUtil.convertToRepresentation(action, Representation.DEFAULT);
 	}
 	
+	private void updateModules(Collection<Module> modules, Collection<String> downloadURLs) {
+		if (modules.isEmpty() || downloadURLs.isEmpty()) {
```

Comment:
Can you use apache commons' StringUtils.isBlank() to catch null and whitespace character?